circuit_knitting/cutting/__init__.py

```diff
@@ -21,6 +21,7 @@
     :toctree: ../stubs/
     :nosignatures:
 
+    find_cuts
```

Comment:
Should this be moved below, under the "Automatic Cut Finding" header?

Reply:
I actually wasn't sure, since it's importable from the `cutting` level, but I do think it should be moved under that header.

Reply:
> I actually wasn't sure, since it's importable from the `cutting` level, but I do think it should be moved under that header.

The things already under that header are importable from the `cutting` level too (which is the way I think it ought to be).